docker/caffe2/jenkins/common/install_pytorch.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+
+pip install numpy
+git clone https://github.com/pytorch/pytorch.git --recursive
+cd pytorch
+git checkout 7fcfed19e7c4805405f3bec311fc056803ca7afb
+pip install -r requirements.txt
+/usr/bin/python3.6 setup.py install
```

Comment:
Can you use `python` here? If you configured Python before this step, you should have the correct Python binary as part of your default PATH.